tests/test_cli.py

```diff
@@ -54,3 +54,19 @@ def test_cli_almanack(tmp_path):
             sorted([metric["name"] for metric in metrics_table["metrics"]]),
         )
     )
+
+
+def test_cli_almanack_check(tmp_path):
```

Comment:
I suppose you are testing the `check` functionality in `test_gather_failed_almanack_metrics` and therefore this test only needs to test the cli portion? Is this true?

It still might be nice to then check a simple return, rather than just return 1 on failures

Reply:
Thanks, yes, this is true on the approach! I'll see about making this test more comprehensive somehow. The challenge is that the string will be unstructured data that is difficult to validate and will change over time.

Reply:
Increased the tests to check for certain strings within the output.